spv/spv.go

```diff
@@ -101,7 +72,7 @@ func validateSatoshisSum(dBeef *DecodedBEEF) error {
 	return nil
 }
 
-func validateLockTime(dBeef *DecodedBEEF) error {
+func validateLockTime(dBeef *paymail.DecodedBEEF) error {
 	if dBeef.ProcessedTxData.LockTime == 0 {
 		for _, input := range dBeef.ProcessedTxData.Inputs {
```

Comment:
What if parent of unmined input has nLockTime? Should we treat it like a problem?
IMHO we should. Otherwise (I assume) if we don't check it someone can try to fool us.

Reply:
done